projects/assets-library/src/images/image-type.ts

```diff
@@ -2,5 +2,10 @@ export const enum ImagesAssetPath {
   ErrorPage = 'assets/images/error-page.svg',
   LoaderSpinner = 'assets/images/loader-spinner.gif',
   LoaderPage = 'assets/images/loader-page.gif',
-  LoaderExpandableRow = 'assets/images/loader-expandable-row.gif'
+  LoaderExpandableRow = 'assets/images/loader-expandable-row.gif',
+  LoaderLogo = 'assets/images/loader-logo.svg',
```

Comment:
These svgs should be loaded via IconModule.

Reply:
Once Migration is done, please remove the loader gifs from here

Reply:
removed gifs and previously added svgs.